pkg/operator/k8sutil/configmap_test.go

```diff
@@ -60,9 +60,10 @@ func TestDeleteConfigMap(t *testing.T) {
 func TestGetOperatorSetting(t *testing.T) {
 	k8s := fake.NewSimpleClientset()
 
+	operatorSettingConfigMapName := "rook-ceph-operator-config"
```

Comment:
How about using a non-ceph name for this test?

Reply:
Fixed.